python_docs_theme/theme.conf

```
@@ -32,4 +32,5 @@ root_name = Python
 root_url = https://www.python.org/
 root_icon = py.svg
 root_include_title = True
+root_icon_alt_text = Python logo
```

Any reason why the two `root_icon*` are not together?
```suggestion
root_icon = py.svg
root_icon_alt_text = Python logo
root_include_title = True
```